os/fs/smartfs/smartfs_procfs.c

```diff
@@ -988,6 +990,24 @@ static ssize_t smartfs_dump_psector(FAR struct file *filep, FAR const char *buff
 	/* Return 0 this doesn't bring any data */
 	return 0;
 }
+
+static size_t smartfs_dump_ext(FAR struct file *filep, FAR char *buffer, size_t buflen)
```

Comment:
can you provide some description for this function

Reply:
Added description